js/analyses/getPageTitleLength.js

```diff
@@ -0,0 +1,14 @@
+/** @module analyses/getPageTitleLength */
```

Comment:
This doesn't have to be a module, we're just retrieving the length of a string which can be done wherever this is modules is now used.